### PR TITLE
add drf e2e cases

### PR DESCRIPTION
**Pull request by Thor-wl**
Signed-off-by: Thor <1187526662@qq.com>

### CODE REVIEW

File: test/e2e/drf.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package e2e
```
Comment (hzxuzhonghu) [inline test/e2e/drf.go:1]: copyright

File: test/e2e/drf.go [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+		job.name = "j1-reference"
+		referenceJob := createJob(ctx, job)
+		err := waitTasksReady(ctx, referenceJob, int(rep))
+		Expect(err).NotTo(HaveOccurred())
+
+		job.name = "j2-drf"
+		job.tasks[0].req = halfCPU
+		backfillJob := createJob(ctx, job)
+		err = waitTasksReady(ctx, backfillJob, int(rep))
```
Comment (hzxuzhonghu) [inline test/e2e/drf.go:34]: Can you elaborate how drf work in this case?

Comment (Thor-wl) [reply test/e2e/drf.go:34]: drf works when a task who requires less resource and to be schedulered will preempt resource of another task who require more resource and is running

File: test/e2e/drf.go [hunk 3]
```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2019 The Volcano Authors.
```
Comment (hzxuzhonghu) [inline test/e2e/drf.go:2]: s/2019/2020

File: test/e2e/drf.go (hunk 2 above)
Comment (hzxuzhonghu) [reply test/e2e/drf.go:50]: add comments

**Pull request merged**